index.js

```diff
@@ -1,16 +1,18 @@
 let _ = require("lodash");
+let P = require("bluebird");
 let lovebotPlayer = require("lovebotplayer");
 let Tournament = require('./model/game/tournament');
 let HttpProxyStrategy = require('./httpproxy');
+let rando = (p, o) => P.resolve(lovebotPlayer.random(p, o));
```

Comment:
I decided to let our implementors not have to deal with knowing they are inside promises, we'll wrap it up for them.